FirebaseAI/Sources/GenerativeAIService.swift

```diff
@@ -30,9 +30,12 @@ struct GenerativeAIService {
 
   private let urlSession: URLSession
 
-  init(firebaseInfo: FirebaseInfo, urlSession: URLSession) {
+  private let aiConfig: FirebaseAIConfig
```

Comment:
This could also be propagated down via `FirebaseInfo`, but I wasn't sure if that was desirable. cc @andrewheard

Reply:
Doesn't seem likely, as we don't do the same with `APIConfig`.